modules/client-configuration/pages/supported-features-oracle.adoc

```diff
@@ -0,0 +1,66 @@
+[[supported-features-oracle]]
+= Supported Oracle Features
+
+
+This table lists the availability of various features on {oracle} clients.
+
+The icons in this table indicate:
+
+* icon:check[role="green"] the feature is available on both traditional and Salt clients
+* icon:times[role="danger"] the feature is not available
+* icon:question[role="gray"] the feature is under consideration, and may or may not be made available at a later date
+* ``Traditional`` the feature is supported only on traditional clients
+* ``Salt`` the feature is supported only on Salt clients
+
+
+[cols="1,1", options="header"]
+.Supported Features on {oracle} Operating Systems
+|===
+| Feature | {oracle}{nbsp}8
+| Client | icon:question[role="gray"] (plain CentOS) / ``Salt``
+| Operating system packages  | ``Salt``
+| Registration | ``Salt``
+| Install packages | ``Salt``
+| Apply patches (requires CVE ID) | ``Salt`` (third-party service required for errata)
+| Remote commands | ``Salt``
+| System package states | ``Salt``
+| System custom states | ``Salt``
+| Group custom states | ``Salt``
+| Organization custom states | ``Salt``
+| System set manager (SSM) | ``Salt``
+| Service pack migration | icon:times[role="danger"]
```

Comment:
Service Pack Migration only exists on SLES, so we should say it does not apply on RHEL, Ubuntu, etc instead of saying "not supported". This probably needs to be updated for other OSs as well.

```suggestion
| Service pack migration | N/A
```